DOC-714: New behaviour when clicking links in readonly mode (#1754)
tinymce/tinymce-docs: release-notes/release-notes56.md

For information on the `name` field, see: [User interface - `style_formats`]({{ site.baseurl }}/configure/editor-appearance/#style_formats).

> ### Changed behaviour when clicking links in `readonly` mode

Author: Maybe?

```suggestion
### Clicking a link in `readonly` mode now opens the URL
```

> Clicking links within the editor when in `readonly` mode will now result in a new tab or window being opened (depending on browser preferences).

Reviewer: ```suggestion
Clicking links when the editor is in `readonly` mode will open the URL in a new tab or window (depending on browser preferences).
```
